### ISSUE TITLE
Final prep work

### ISSUE THREAD

**Issue by WasabiFan** [kyranstar/Hunt-The-Wumpus#52]
- Make the camera slightly more zoomed in by default
- Make the Wumpus disappear when appropriate
- Run through the whole game to look for issues

**Comment by WasabiFan** [comment]
Once we get the other main things done, we should tick these off the list. This could probably be finished day-of if needed.

**Comment by kyranstar** [comment]
Should the wumpus appear when it moves into explored territory? I think it makes it too easy. This applies to bats & pits as well.

**Comment by WasabiFan** [comment]
Yeah... I think these things should probably disappear when not in the immediate vicinity. Maybe they should fade out over a long time, say 10 or 20 seconds?

On this topic, it would probably be a good idea to refactor the `DrawRoomBase` method into separate methods:
- `DrawRoomBase` should just draw the base images -- the hexagons and their coloring
- We should have a separate method for hazards (pits, bat)
- And another for the player/Wumpus

Then we can easily modify it from there to hide them or fade them. @kyranstar Are you willing to try and tackle this?

**Comment by kyranstar** [comment]
Sure.

**Comment by kyranstar** [comment]
@WasabiFan Do you mean DrawCaveBase?

**Comment by WasabiFan** [comment]
Yeah... that sounds right. I was going off (apparently incorrect) memory ;)

**Comment by anak4569** [comment]
Guys, do we have a set of bookmarked code pieces to showcase?

**Comment by WasabiFan** [comment]
No... although we should. If someone can find some of those, that would be great!

P.S. Did you get a chance to add the trivia answer order randomization?

**Comment by anak4569** [comment]
I thought I did, the questions answer's aren't all the first choice now.

**Comment by WasabiFan** [comment]
Looking at your commit, I don't see that -- are you talking about the `StandbySet`? Those aren't actually used, so changing it there won't change the available answers (sorry if I wasn't very clear on that) I'll go there and quickly rejigger that stuff to make it work with the actual questions.

Thanks for doing most of it!

**Comment by WasabiFan** [comment]
Ahhh I see what you did -- you then set the question set to the standby set. I'll go and move that around to work with the other system.

**Comment by anak4569** [comment]
Chill legumes.

**Comment by WasabiFan** [comment]
I... don't know what that means :confused: I'm not familiar with the lingo of today's modern youth...
